clients/include/aux/testing_set_get_matrix.hpp

```diff
@@ -54,7 +54,7 @@ inline hipblasStatus_t testing_set_get_matrix(const Arguments& arg)
     // memory
     if(rows < 0 || cols < 0 || lda <= 0 || ldb <= 0 || ldc <= 0)
     {
-        return HIPBLAS_STATUS_INVALID_VALUE;
+        return;
     }
```

Comment:
Maybe need a ticket or TODO as in the future this should be a bad_arg test or report arguments for which no test was run so the yaml can be cleaned.

Reply:
Created LWPMLSE-504 for bad_arg tests in hipblas.